app/assets/stylesheets/email.css.scss

```diff
@@ -39,9 +39,13 @@
   line-height: 60px;
 }
 
-h4 {
-  font-weight: bold;
-  margin-bottom: 16px;
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  line-height: 1.4;
```

Comment:
Noting that this moves us to consistency with the design system and not the IdP's current typography, since in the IdP we currently use [`1.35` for H1 and H2](https://github.com/18F/identity-idp/blob/e44cfbb4d398baa2c786d36c228be409afb8fb53/app/assets/stylesheets/utilities/_typography.scss#L19-L35), and [`1.5` for every other heading level](https://github.com/18F/identity-idp/blob/e44cfbb4d398baa2c786d36c228be409afb8fb53/app/assets/stylesheets/utilities/_typography.scss#L37-L71). 

I don't feel too strongly about this and I think we'll want to eventually converge on a standard specification, but my understanding from [the discussion yesterday](https://gsa-tts.slack.com/archives/C046P4WEJDC/p1694536900952179?thread_ts=1694534191.438749&cid=C046P4WEJDC) was that we were going to be aligning to the IdP's current typography for now.

Reply:
Reviewing the conversation, I never saw a reference to _**current**_ desktop headings/typography from Nick nor Allison. Therefore I went with the design system. I expect this will also help reduce rework when the changes are finalized.